src/lyapunov_spectrum.jl

```diff
@@ -3,26 +3,6 @@ export lyapunovspectrum!, lyapunovspectrum
 const δqind = SV{Int}(1,2)
 const δpind = SV{Int}(3,4)
```

Comment:
We have to do type-stability checks for this. I think it is fine but it isbest to check!